src/utils/settings.ts

```diff
@@ -37,7 +37,10 @@ export const networkEndpoints: Record<Exclude<NetworkType, 'custom'>, Settings>
   }
 }
 
-export const walletIdleForTooLongThreshold = 3 * 60 * 1000 // 3 minutes
+const lockWalletAfter = 3 * 60 * 1000 // 3 minutes
+const lockWalletAfterDev = 9999999999
```

Comment:
In general, I like to have development conditions close to the final release version, so that I'm doing proper QA while developing. 
But if you find the auto-lock too annoying, we should add a test for it to make sure it's not broken and we don't see it.

Reply:
Actually, every time I change something I need to refresh the app, so it locks me out anyway... I'll remove this commit.

Reply:
done

Reply:
but I will add tests eventually as well ;)

Reply:
Yeah, the inability to interact with the app after a change is really annoying - this I agree. It's linked to the CSP, not allowing React to inject new code, even while developing.

If you are motivated, it would be very nice to find how to :
1) Either adapt the CSP to keep it safe but allow code injection for a local source (self).
2) Or remove the rule, but only in dev mode.

Reply:
Yes, I noticed the CSP issue. For now I just added `unsafe-inline` without committing it. I'll look for a proper solution.

Reply:
Until then, I think we can merge this one!